cloud-formation/subscriptions-app.cf.yaml

```diff
@@ -180,6 +180,12 @@ Resources:
           - Effect: Allow
             Action: sqs:*
             Resource: arn:aws:sqs:eu-west-1:865473395570:subs-holiday-suspension-email*
+      - PolicyName: InvokeAVLambda
+        PolicyDocument:
+          Statement:
+          - Effect: Allow
+            Action: sts:AssumeRole
+            Resource: arn:aws:iam::021353022223:role/support-invoke-value-calculator
```

Comment:
it's good practice not to commit the arn

Reply:
That's not the convention in this file though